server/src/main/java/org/elasticsearch/index/IndexSettings.java

```diff
@@ -283,7 +283,7 @@ public final class IndexSettings {
         TimeValue.MINUS_ONE,
         Property.NodeScope
     ); // TODO: remove setting
-    public static TimeValue STATELESS_DEFAULT_REFRESH_INTERVAL = TimeValue.timeValueSeconds(10); // TODO: settle on right value
+    public static TimeValue STATELESS_DEFAULT_REFRESH_INTERVAL = TimeValue.timeValueSeconds(15); // TODO: this value is still not final
```

Comment:
I guess it makes sense to update the PR description to reflect that the actual value is 15s

Reply:
Thanks, updated it now.